docs/user/resources/04-100-10-aws-nfs-volume-restore.md

```diff
@@ -0,0 +1,51 @@
+# AwsNfsVolumeRestore Custom Resource
+
+The `awsnfsvolumerestore.cloud-resources.kyma-project.io` namespaced custom resource (CR) describes the AWS EFS file 
+system full restore operation on the same existing EFS file system.
+
+Out-of-place restore and Item-Level restore are not supported at this version of the `Cloud Manager`.
+
+While the EFS file system restore operation is running in the underlying cloud provider subscription, it needs its 
+source backup and its destination file system to be available. Upon its completion, restored file system 
```

Comment:
```suggestion
source backup and its destination file system to be available. When the restore operation is finished, the restored file system is available on a recovery directory, named `aws-backup-restore_{datetime}`, off of the root directory.
```
File paths get [code font](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/03-formatting.md#items-written-in-code-font) :)

Reply:
What does "off of" mean? Is the recovery directory a child or sibling of the root directory?

Reply:
> What does "off of" mean? Is the recovery directory a child or sibling of the root directory?

AWS documents use this expression so I used it here. It means child in this case.